web/src/config/client_swagger.json

```diff
@@ -10,40 +10,22 @@
 		"/product_version": {
 			"get": {
 				"tags": ["About"],
-				"summary": "Gets version",
+				"summary": "Gets product version and its components",
 				"responses": {
 					"200": {
-						"description": "OK"
-					},
-					"201": {
-						"description": "No Content"
+						"description": "OK: Product version and components fetched successfully."
 					},
 					"400": {
-						"description": "Bad Request"
-					},
-					"401": {
-						"description": "Unauthorized"
-					},
-					"403": {
-						"description": "Forbidden"
-					},
-					"404": {
-						"description": "No Data Found"
-					},
-					"409": {
-						"description": "Conflict"
+						"description": "Bad Request: Invalid request message received."
 					},
 					"422": {
-						"description": "Unprocessable Entity"
+						"description": "Unprocessable Entity: server understands the content type of the request entity, and the syntax of the request entity is correct, but it was unable to process the contained instructions."
```

Comment:
This can be removed , as we are not sending any body with request, it will not occur

Reply:
This error code "422" removed